docs/api-style-guide.md

```diff
@@ -0,0 +1,11 @@
+# GruppOne's API style guide
+
+<!-- TODO write this docs file! -->
+<!-- XXX should this be in ita or en? -->
```

Comment:
Direi in italiano. Una regola potrebbe essere che va tutto in inglese tranne la cartella docs. In questo modo la documentazione che ci mettiamo dentro poi può essere riciclata per i manuali.